xla/service/gpu/gpu_offloading_test.cc

```diff
@@ -142,6 +147,48 @@ ENTRY %main (param_0: f32[1024], param_1: f32[1024]) -> f32[1024] {
   EXPECT_TRUE(RunAndCompare(hlo_text, ErrorSpec{/*aabs=*/1e-6, /*arel=*/1e-6}));
 }
 
+TEST_F(GpuCompilerTest, FusedComputationTest) {
```

Comment:
Could we do a better test name?

Reply:
Yes, change it to OffloadFusedComputationTest().